openwisp_controller/config/tests/test_admin.py

```diff
@@ -1259,16 +1259,16 @@ def test_admin_menu_groups(self):
         self.client.force_login(self._get_admin())
         response = self.client.get(reverse('admin:index'))
         for model in models:
-            with self.subTest(f'test_menu_group_for_{model}_model'):
+            with self.subTest(f'test menu link for {model} model'):
```

Comment:
Hi,
Can you please make this change ``test menu link`` > ``test menu group link``. I am making this change in each module because makes more sense.

Reply:
I get the point, the function that register the menu item is called "menu group"